### PR TITLE
[DX-1051] Fix: SDK switcher styling

### PR DESCRIPTION
**Pull request by CodeSchwert**
## Description

- Re-enable the SDK switcher component and fix the styling (remove Tailwind CSS and add manual/custom CSS).
- Remove Tailwind CSS from the project.

## Resolved issues

Jira: https://immutable.atlassian.net/browse/DX-1051

### Before submitting the PR, please consider the following:
- [x] It's beneficial if your pull request references an issue used to discuss the pull request ahead of time. If you haven't previously created an issue, please create one and discuss your contribution with the maintainers.
- [x] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [x] The description should clearly illustrate what problems the pull request solves.
- [x] Ensure that the commit messages follow our guidelines.
- [x] Resolve merge conflicts (if any).
- [x] Make sure that the current branch is up to date with the `main` branch.

### For internal Immutable developers:
- [ ] If you are adding or updating documentation for an SDK, please make sure that you meet the requirements in [this doc](https://immutable.atlassian.net/wiki/spaces/PPS/pages/1916994017/SDK+documentation+guide).
- [ ] Please obtain PR approval from all of the following:
    - Your tech lead (Fallback: Team lead or engineering manager)
    - Your product manager (Fallback: Group PM)
    - Another software engineer on your team

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -1,6 +0,0 @@
-// module.exports = {
```
Comment (matthewmuscat) [inline postcss.config.js:1]: Can we just remove this if we're removing other tailwind resources?

File: src/components/SdkSwitcher/styles.module.css [hunk 2]
```diff
@@ -1,8 +1,152 @@
...
+.switcherLabel {
+  padding-left: 0.25rem;
+  font-weight: bold;
+  font-size: small;
+  color: var(--ifm-menu-color);
+}
+
+.switcherButton {
+  box-sizing: border-box;
+  border: 0 solid #e5e7eb;
```
Comment (matthewmuscat) [inline src/components/SdkSwitcher/styles.module.css:15]: Can we utilise a colour variable here?

File: src/components/SdkSwitcher/styles.module.css [hunk 3]
```diff
@@ -1,8 +1,152 @@
...
+  margin: 0;
+  position: relative;
+  width: 100%;
+  cursor: default;
+  border-radius: 0.375rem;
+  border-width: 1px;
+  padding-bottom: 0.5rem;
+  padding-top: 0.5rem;
+  padding-left: 0.75rem;
+  padding-right: 2.5rem;
```
Comment (matthewmuscat) [inline src/components/SdkSwitcher/styles.module.css:28]: You could probably just make this `padding: 0.5rem 2.5rem 0.5rem 0.75rem;` to reduce line count.

File: src/components/SdkSwitcher/styles.module.css [hunk 4]
```diff
@@ -1,8 +1,152 @@
...
+  border: 1px solid #e5e7eb;
+}
+
+/* li */
+.switcherListOption[aria-expanded='true'] {
+  word-wrap: break-word;
+  box-sizing: border-box;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```
Comment (matthewmuscat) [inline src/components/SdkSwitcher/styles.module.css:95]: This is here yet [this](https://github.com/immutable/imx-docs/pull/67/files#diff-cfa6914be7dfc92433290b802b6983e327d13f27e5e57ad89c158e5429617794R103-R104) is also being applied? 

Could we just make it `padding: 0.25rem 0;`

**Review by matthewmuscat — CHANGES_REQUESTED** [review]

File: postcss.config.js (hunk 1 above)
Comment (CodeSchwert) [reply postcss.config.js:1]: I'll check this, I thought it was removed.

Comment (matthewmuscat) [reply postcss.config.js:1]: Oh yep, it is sorry. Github issue

**Review by matthewmuscat — APPROVED** [review]

**Review by ckailash — APPROVED** [review]
:shipit:

**Pull request merged**